inc/defaults/global_merge_tags.php

```diff
@@ -134,3 +134,36 @@
 		)
 	)
 );
+
+notification_add_global_merge_tag(
+	new MergeTag\DateTime\Date(
+		array(
+			'slug'      => 'date',
+			'name'      => __( 'Current date', 'notification' ),
```

Comment:
I'd like you to adjust the name, because it's not clear what `current` means. Current to editing time?
```suggestion
			'name'      => __( 'Trigger execution date', 'notification' ),
```